### ISSUE TITLE
Dashboard errors if pod's owner reference is not supported

### ISSUE THREAD

**Issue by yqwang-ms** [kubernetes/dashboard#3251]
##### Steps to reproduce

The pod is created as:
`podObject.OwnerReferences = []meta.OwnerReference{*meta.NewControllerRef(configMapObject, ConfigMapGroupVersionKind)}`

##### Environment
```
Installation method: 
Kubernetes version:
Dashboard version: v1.8.1
Commit: 7b3553aecf28224ef7275e157b5bba7dd23f436a
```

##### Observed result
Dashboard reported Internal Server Error (500):
```
Unknown reference kind ConfigMap

```

##### Comments

By setting the owner is also a controller, the Pod functions normally, except for its detail cannot be viewed from kube dashboard, and the dashboard shows error:
"Unknown reference kind ConfigMap".

Seems this is related to https://github.com/kubernetes/dashboard/issues/3158

Please help :)

**Comment by maciaszczykm** [comment]
We have to solve it because there are multiple resources that are not supported.

**Comment by maciaszczykm** [comment]
@yqwang-ms If you are familiar with Go, perhaps you would be interested in fixing this issue so we can release fix for it :) We are quite occupied with Angular migration and we do not have time for it at the moment.

**Comment by yqwang-ms** [comment]
Thanks @maciaszczykm for the comment, I will try first but not sure :)

**Comment by yqwang-ms** [comment]
Hi @maciaszczykm I have not got free to look into the code, do you have resource to fix it now? :)

**Comment by maciaszczykm** [comment]
At the moment I am focused on getting ng migration done. Perhaps someone else will pick it up.

**Comment by mrow4a** [comment]
@maciaszczykm is this planned to be included in next release, or earliest 2.0.0? It is tiny fix, but would be great to have it for these who already use CRDs early (https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/issues/484).

**Comment by maciaszczykm** [comment]
Earliest in 2.0.0, @mrow4a. If you would be willing to create bugfix branch on top of old release then we can get it released perhaps.
